src/vs/workbench/contrib/chat/browser/chatStatus.ts

```diff
@@ -505,13 +505,15 @@ class ChatStatusDashboard extends Disposable {
 			} else {
 				usedPercentage = Math.max(0, 100 - quota.percentRemaining);
 			}
+			// Use intl number format to format the used percentage to 1 decimal place
+			const percentUsedFormatter = new Intl.NumberFormat(undefined, { maximumFractionDigits: 1, minimumFractionDigits: 0 });
 
 			if (quota.unlimited) {
 				quotaValue.textContent = localize('quotaUnlimited', "Included");
 			} else if (quota.overageCount) {
 				quotaValue.textContent = localize('quotaDisplayWithOverage', "+{0} requests", quota.overageCount);
```

Comment:
do we want to have the same formatting for overage?

Reply:
I wanted to minimize the change, and since overage comes directly from the limiter I don't think we will have this massive decimal overflow issue that percentage has. I'll add it though just to be safe

Reply:
Fyi I will port that part of the change over to `main`